### DISCUSSION TITLE
Can we simplify the Openverse Make site handbook and defer to the documentation site almost exclusively?

### DISCUSSION THREAD

**Discussion by sarayourfriend** [WordPress/openverse#1241] [Q&A] [answered]
I doubt we can get rid of the Make handbook entirely (though perhaps @zackkrida could find out for us), but needing to maintain two separate locations of documentation, and remember what is where (there's so much catalog documentation on the handbook, but all of that would be awesome to have in the documentation site), doesn't feel like a good long-term solution.

It is not desirable to move all our documentation into the make site. Make is harder for us to iterate on. Can we move nearly all the documentation out of Make and into the documentation site instead?

**Comment by dhruvkb** [comment]
There is a lot of content is the Make handbook that's oriented towards developers. That definitely should be moved to the docs site.

However the Make handbook contains some pages like "History of Openverse" and "2022 Roadmap" that seem like a better fit for the handbook, considering that these pages mostly for people outside (and above) the team and the content in them is either static, or doesn't change very often.

Maybe we can keep both and define rules for what type of documentation goes where. I see pages that pertain to a _somewhat political nature_ being a better fit for the Make handbook.

**Comment by zackkrida** [comment, marked as answer]
> Can we move nearly all the documentation out of Make and into the documentation site instead?

Yes, absolutely! This was discussed by my cohort during the docs bash, I proposed it there, and we all gave it a 👍. Here's the line from the spreadsheet: https://docs.google.com/spreadsheets/d/1AW5LAzzqRhG_oRKFojCe4frV7ASdDTtVdQHhkbpVDQU/edit#gid=0&range=20:20

The specific tasks I outlined there were:

> 
> Dramatically reduce and remove outdated content in the Openverse handbook:
> - Keep the Welcome page:
>   - Add links to the developer docs and the contribution guide.
> - Keep Openverse and Photo Directory
> - Keep Development Weekly Chat Facilitation Guide
> - Move "history" to the about page on openverse.org or to a new page on openverse.org
> - Delete "New contributor guide"; link to external doc
> - Remove the "Openverse Frontend" page
> - Move "Random tips for API testing, local development, etc." into the developer docs
> - Move the "Openverse Catalog" section into the developer docs site

For SEO reasons it makes sense to move some non-technical Openverse backstory into openverse.org itself.
